modules/currency-l0/src/main/scala/org/tessellation/currency/l0/cli/method.scala

```diff
@@ -142,6 +144,45 @@ object method {
     }
   }
 
+  case class RunGenesisWithSigning(
```

Comment:
So just to be clear from the earlier comment, I'm not sure this mode actually needs to be supported (but it may actually be useful from user perspective so they can use a single command to both build the genesis and sign things.)

The main command we need to support right now is just a single CLI command RunFeeStakingSigning which only accepts those two parameters, and outputs a signed blob corresponding to the keys loaded from the conventional wallet environment variable paths.